print/print-apps/oereb/toc.jrxml

```diff
@@ -306,7 +307,7 @@
 				<textElement verticalAlignment="Top" markup="html">
 					<font fontName="Cadastra" size="6"/>
 				</textElement>
-				<textFieldExpression><![CDATA[$R{FurtherInformationContent}]]></textFieldExpression>
+				<textFieldExpression><![CDATA[$P{Further_Information_Text}]]></textFieldExpression>
```

Comment:
I prefer camel case...
Actually we use the _ in the print attributes when we flatten the attributes of an object `<Object>_<Attribute>`.